app/controllers/admin/user_creations_controller.rb

```diff
@@ -1,46 +1,69 @@
 class Admin::UserCreationsController < ApplicationController
   
   before_filter :admin_only
+  before_filter :get_creation
```

Comment:
Not desperately important 

However all before_filters syntax are deprecated in Rails 5.0 and will be removed in Rails 5.1 ( http://stackoverflow.com/questions/16519828/rails-4-before-filter-vs-before-action )